src/Components/SetupConnectScreen/ConnectSuccessfulStep.tsx

```diff
@@ -32,7 +32,7 @@ export class ConnectSuccessfulStep extends React.Component<Props, State> {
 
     render(): React.ReactNode {
         const description = <Typography variant="subtitle1" component="h1" gutterBottom sx={{marginTop: '8px'}}>
-            Congrats! You've successfully created an <code>access_token</code>, which allows you to fetch data for the User you just created.
+            After generating an <code>access_token</code>, you can now fetch data for the User you created.
```

Comment:
This wording makes it seem like they still seem to create an access_token.

I'd prefer "You've generated an access_token and can now..."